core/src/main/java/org/mapfish/print/map/image/wms/WmsUtilities.java

```diff
@@ -69,6 +69,9 @@ public static URI makeWmsGetLayerRequest(final MfClientHttpRequestFactory reques
 
         if (wmsLayerParam.serverType != null && dpi != Constants.PDF_DPI) {
             addDpiParam(extraParams, (int) Math.round(dpi), wmsLayerParam.serverType);
+            int width = (int) (imageSize.width * dpi / Constants.PDF_DPI);
+            int height = (int) (imageSize.height * dpi / Constants.PDF_DPI);
+            getMapRequest.setDimensions(width, height);
```

Comment:
@tsauerwein Strange. When I tested it did have an impact. Maybe something changed elsewhere that would explain it. Moreover, even if I revert my patch, the height and width are correctly updated now, the presentation looks correct and the problem I describe in #449 is solved.